tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -720,12 +721,18 @@ export class WebGPUBackend extends KernelBackend {
       if (preLength === 5 || preLength === 6) {
         baseAlignment = 16;
       }
+      if (baseAlignment > maxAlignmentOfField) {
+        maxAlignmentOfField = baseAlignment;
+      }
       currentOffset = Math.ceil(currentOffset / baseAlignment) * baseAlignment;
       preLength = d.data.length;
       offsets.push(currentOffset);
       currentOffset += d.data.length * 4;
     });
 
+    currentOffset = currentOffset % maxAlignmentOfField === 0 ?
+        currentOffset :
+        Math.ceil(currentOffset / maxAlignmentOfField) * maxAlignmentOfField;
```

Comment:
Why not directly use `Math.ceil(currentOffset / maxAlignmentOfField) * maxAlignmentOfField;`